tests/testthat/_snaps/ratio.md

```diff
@@ -23,7 +25,8 @@
     Condition
       Error in `step_ratio()`:
       Caused by error in `prep()`:
-      ! All columns selected for the step should be double, or integer.
+      x All columns selected for the step should be double or integer.
+      * 8 factor variables found: `x5`, `x5`, `x5`, `x5`, ...
```

Comment:
What's happening here?

Reply:
solved with a call to `unique()` [`0a36bb7` (#1235)](https://github.com/tidymodels/recipes/pull/1235/commits/0a36bb71440d49b5d96df6f481e6d367bee77418)